### PR TITLE
Added objects for backend restapi/firebase compatibility

### PR DESCRIPTION
**Pull request by xumaple**

### CODE REVIEW

File: setup/rest-api-docs.md [hunk 1]
```diff
@@ -177,7 +177,7 @@ Response:
     "qid": "my_question_id1",
     "aid": "my_answer_id1",
     "time": <TimeFormatTBD>, 
-    "responses-left": <int>
```
Comment (shlee2017) [inline setup/rest-api-docs.md:180]: you think responses_left is better cause of consistency? like we have my_question_id

Comment (xumaple) [reply setup/rest-api-docs.md:180]: All underscores you see are in values not keys. "my_question_id1" is just a placeholder for whatever the id will end up being.

There will be no snake case - the default for javascript is camelcase.

File: server/src/app.dtos.ts [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  time: Time,
+  qid: QID
+}
+
+export interface getAnswerListOutDto {
+  aids: Answer[]
+}
+
+export interface getAnswerInDto {
+  aids: AID[]
```
Comment (orccrusher99) [inline server/src/app.dtos.ts:50]: do we need question id here too

Comment (xumaple) [reply server/src/app.dtos.ts:52]: Added

**Pull request merged**